packages/pages/src/build/build.ts

```diff
@@ -1,17 +1,15 @@
-import shell from "shelljs";
 import { CommandModule } from "yargs";
 import { ProjectFilepaths } from "../common/src/project/structure.js";
+import { build } from "vite";
 
 type BuildArgs = Pick<ProjectFilepaths, "scope">;
 
 const handler = async ({ scope }: BuildArgs) => {
   // Pass CLI arguments as env variables to use in vite-plugin
-  shell.exec("vite build", {
-    env: {
-      ...process.env,
-      YEXT_PAGES_SCOPE: scope,
-    },
-  });
+  if (scope) {
+    process.env.YEXT_PAGES_SCOPE = scope;
```

Comment:
Out of scope but it feels a little strange to me that we passed scope as an env variable here

Reply:
I poked around a little, but couldn't find another way to pass along CLI arguments if they weren't only meant to impact the vite config